tests/tests/test-fees/test-fee-multiplier.ts

```diff
@@ -425,3 +426,34 @@ describeDevMoonbeam("Fee Multiplier - XCM Executions", (context) => {
     expect(initialValue.eq(postValue), "Fee Multiplier has changed between blocks").to.be.true;
   });
 });
+
+describeDevMoonbeam("TransactionPayment Runtime Queries", (context) => {
+  it("should be able to query length fee", async function () {
+    const adjusted_length_fee =
+      await context.polkadotApi.call.transactionPaymentApi.queryLengthToFee(1n);
+    expect((adjusted_length_fee as any).toBigInt()).to.eq(1_000_000_001n);
+  });
+
+  it("should be able to query weight fee", async function () {
+    const adjusted_weight_fee =
+      await context.polkadotApi.call.transactionPaymentApi.queryWeightToFee({
+        refTime: 1,
+        proofSize: 1,
+      });
+    expect((adjusted_weight_fee as any).toBigInt()).to.eq(50_000n);
```

Comment:
You can replace this assertion value here is WEIGHT_FEE defined in [constants.ts](https://github.com/PureStake/moonbeam/blob/d303c973bc92ced8ba9d2d04aaa436616cf495a9/tests/util/constants.ts#L115) if you feel this is appropriate.

If not, feel free to define some constants in there with appropriate names which you want to assert against (which can then be reusable in other tests).